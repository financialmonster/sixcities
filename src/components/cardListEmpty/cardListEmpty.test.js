import React from 'react';
import renderer from 'react-test-renderer';

import CardListEmpty from './cardListEmpty.jsx';

it(`Correctly renders the CardListEmpty component`, () => {
    const tree = renderer.create(<CardListEmpty city='Paris' />)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
