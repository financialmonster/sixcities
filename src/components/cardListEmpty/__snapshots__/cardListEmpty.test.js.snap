// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Correctly renders the CardListEmpty component 1`] = `
<div
  className="cities__status-wrapper tabs__content"
>
  <b
    className="cities__status"
  >
    No places to stay available
  </b>
  <p
    className="cities__status-description"
  >
    We could not find any property available at the moment in 
    Paris
  </p>
</div>
`;
